graphql-server/src/routes/auth/post-login.ts

```diff
@@ -0,0 +1,24 @@
+import { REACT_SERVER_ORIGIN } from "../../utils/constants";
+
+/**
+ * Upon successful login, the user is redirected to this route, which
+ * closes the popup window and sends user's email to the parent window
+ * to display on in the navbar.
+ */
+export function postLoginRouter(req: any, res: any) {
+  const userEmail = req.user.email;
+
+  res.send(`
```

Comment:
this seems like code that should live in the frontend.  servive should just respond with some data indicating what to do.

Reply:
Thanks, Aaron. Heads up that I'm moving this work into [another ticket](https://app.zenhub.com/workspaces/metadb-scrum-5fcfa28a4ccc1600165347b6/issues/gh/mskcc/smile-server/1086) to limit the scope of this PR to just the `graphql-server`.